charts/bookkeeper-operator/templates/operator.yaml

```diff
@@ -26,8 +26,12 @@ spec:
           name: metrics
         command:
         - bookkeeper-operator
+        args:
         {{- if .Values.testmode.enabled }}
-        args: [-test]
+        - -test
+        {{- end }}
+        {{- if .Values.disableFinalizer }}
+        - -disableFinalizer
```

Comment:
Could u add a check to ensure that `args` is added only if `testmode.enabled = true` or `disableFinalizer = true` and not otherwise; in case of both operators?